images/src/tools.ts

```diff
@@ -0,0 +1,39 @@
+import { analyzeImages } from "./analyze.ts";
+import { generateImages } from "./generate.ts";
+
+if (process.argv.length !== 3) {
+    console.error('Usage: node tool.ts <command>')
+    process.exit(1)
+}
+
+const command = process.argv[2]
+
+try {
+    switch (command) {
+        case 'analyzeImages':
+            analyzeImages(
+                process.env.PROMPT,
+                // Split the string into an array of image URLs, while being careful to handle URLs that contain commas.
+                process.env.IMAGES?.split(/(?<!\\),/).map(image => image.replace(/\\,/g, ',')),
+                parseInt(process.env.MAX_TOKENS ?? '2048'),
+            )
+            break
+        case 'generateImages':
+            generateImages(
+                process.env.PROMPT,
+                process.env.MODEL,
```

Comment:
It doesn't look this this is exposed as a `Param:` for this tool. Should we add it?

Reply:
Not sure users should be able to change the model yet since some models don't support vision.

Reply:
There's decent vision support out there, its really image-generation that's lacking.

Without a param for model, this is a regression for our support of vision with the ollama-provider.

Reply:
Got it. I'll add the model param back to the tool.

Reply:
done